test/request_source/stub_plugin_impl.h

```diff
@@ -0,0 +1,55 @@
+// Implementations of RequestSourceConfigFactory and the RequestSources that those factories make.
```

Comment:
We probably want to say something else here, i.e. we should talk about the stub plugin which is implemented in this file.